README.md

```diff
@@ -99,8 +106,10 @@ A target can have the following properties:
   * `host`: the ip
   * `port`: the port
   * `type`: 'wifi' or 'file'
-  * `status`: 'replication-started', 'media-connected', 'osm-connected', 'replication-error'
-  * `message`: any extra information needed for this target, a message from mapeo-sync
+  
+### sync.on('target', target)
+
+Emitted when a new wifi target is discovered.
 
 ### var ev = sync.syncToTarget(target)
```

Comment:
* what do you think about calling this `sync.start(target)`?
* would it be possible to implement a `sync.stop(target)` to cancel the sync?

Reply:
(by the way, not saying to implement `sync.stop` here, but just wondering if it would be possible with current arch)

Reply:
Good Q! I think it's safe to interrupt the sync stream; maybe just doing `stream.destroy()` would be OK?

Reply:
I'm good with the name change